clients/ts/common/package.json

```diff
@@ -10,10 +10,10 @@
   "author": "",
   "license": "ISC",
   "devDependencies": {
-    "@types/jest": "^22.2.3",
+    "@types/jest": "^23.3.2",
     "@types/node": "^8.5.2",
     "@types/webpack": "^4.4.0",
-    "jest": "^22.4.3",
+    "jest": "^23.6.0",
```

Comment:
FYI @Eilon. Updated `devDependencies` (internal use only, not distributed)

Reply:
Up-roved.